notification.py

```diff
@@ -622,8 +626,52 @@ def notify(self, pokemon, time_of_day):
                     n=name, s=mean))
                 return False
 
-        notified = Notification(pokemon_id, coordinates, time_till_hidden, iv, moves, iv_score, time_of_day).notify()
-        if notified:
+        whpushed = False
+        if config.WEBHOOKS:
+            whpushed = self.webhook(pokemon, time_till_hidden)
+
+        notified = False
+        if config.PB_API_KEY or config.TWITTER_ACCESS_KEY:
+            notified = Notification(pokemon_id, coordinates, time_till_hidden, iv, moves, iv_score, time_of_day).notify()
+
+        if notified or whpushed:
             self.last_notification = monotonic()
             self.recent_notifications.append(encounter_id)
-        return notified
+        return notified or whpushed
+
+    def webhook(self, pokemon, time_till_hidden):
+        """ Send a notification via webhook
+        """
+        if isinstance(time_till_hidden, (tuple, list)):
+            time_till_hidden, _ = time_till_hidden
+
+        data = {
+            'type': "pokemon",
+            'message': {
+                "encounter_id": pokemon['encounter_id'],
+                "pokemon_id": pokemon['pokemon_id'],
+                "last_modified_time": pokemon['seen'],
+                "spawnpoint_id": pokemon['spawn_id'],
+                "latitude": pokemon['lat'],
+                "longitude": pokemon['lon'],
+                "disappear_time": pokemon['seen'] + time_till_hidden,
+                "time_until_hidden_ms": time_till_hidden,
+                "individual_attack": pokemon.get('individual_attack', 0),
+                "individual_defense": pokemon.get('individual_defense', 0),
+                "individual_stamina": pokemon.get('individual_stamina', 0),
+                "move_1": pokemon.get('move_1', 0),
+                "move_2": pokemon.get('move_2', 0)
```

Comment:
@sLoPPydrive That's weird. Why would they say that a Pokémon with unknown IVs is 0/0/0? A Pokémon could really have 0/0/0, so how are you supposed to know if it's unknown or really 0?

Anyways, I think these should be changed to just plain `.get()` so that they fall back on `None` for software that doesn't do strange things.